content/applications/sales/crm/acquire_leads.rst

```diff
@@ -6,7 +6,6 @@ Acquire leads
    :titlesonly:
 
    acquire_leads/convert
-   acquire_leads/generate_from_email
```

Comment:
I forgot to mention this:

When you move/remove/rename documentation, you should edit the `redirects.txt` file
You can find it under the main `documentation-user` folder. Instructions are given at the beginning of the file and it's relatively easy. You just need to add a line for each redirection using this format:
`file_path/old_file.rst file_path/new_file.rst # (#PRnumber)`

Here you have moved the content from the `generate_from_email` page to the `generate_from_website` page, meaning we should redirect `generate_from_email` to `generate_from_website` from now on. The following line should be added to the redirects.txt

```
content/applications/sales/crm/acquire_leads/generate_from_email.rst content/applications/sales/crm/acquire_leads/generate_from_website.rst # (#986)
```

Reply:
Okay, so I'm not really sure where in the redirects file I am supposed to put this. I'm guessing that it would go in the "redirections introduced in the 14.0" section, right?

Reply:
Yes indeed, you should put the redirects at the end of the file!